.github/workflows/test-dotnet.yml

```diff
@@ -32,6 +35,11 @@ jobs:
         run: ls -lah
         working-directory: ${{ env.WORKING_DIR }}
 
+      - name: Start MongoDB
```

Comment:
This is *really* premature. You need to document all technical decisions you're taking. If you've chosen MongoDB, you need to say why and what is the state of the art in this area. But, as I say in #35, you need to organize development in progressively complex products; you can't even think about storage until you've designed and tested all relevant business logic.
Also, anything related to data storage needs to be mocked for testing. No need to start a real database for unit tests.